AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py

```diff
@@ -0,0 +1,66 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+# Test Case Title : Check that the basic client connect test works
+
+
+# fmt: off
+class Tests():
+    enter_game_mode =           ("Entered game mode",                       "Failed to enter game mode")
+    exit_game_mode =            ("Exited game mode",                        "Couldn't exit game mode")
+    find_network_player =       ("Found network player",                    "Couldn't find network player")
+# fmt: on
+
+
+def Multiplayer_BasicConnectivity_Connects():
+    r"""
+    Summary:
+    Runs a test to make sure that a player can be spawned
+
+    Level Description:
+    - Dynamic
+        1. Although the level is empty, when the server and editor connect the server will spawn the player prefab.
+    - Static
+        1. This is an empty level.
+                    
+    Expected Outcome:
+    We should see the player connect and spawn with no errors.
+    
+    :return:
+    """
+    import azlmbr.legacy.general as general
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import Tracer
+
+    from editor_python_test_tools.utils import TestHelper as helper
+
+
+    level_name = "BasicConnectivity_Connects"
```

Comment:
nit: If var is constant I would use either use ALL_CAPS or use : Final (if we support Py 3.8) to indicate it should be not be reassigned.